utils/controleval.py

```diff
@@ -99,13 +99,17 @@ def create_parser():
     subparsers = parser.add_subparsers(dest="subcmd", required=True)
     statsparser = subparsers.add_parser(
         'stats',
-        help="outputs statistics for the given benchmark and level.")
-    statsparser.add_argument("-i", "--id", dest="id", help="id of the controls file.",
-                             required=True)
-    statsparser.add_argument("-l", "--level", dest="level", help="level to display statistics of.",
-                             required=True)
-    statsparser.add_argument("-p", "--product", type=str,
-                             help="Product to check has required references")
+        help="calculate and return the statistics for the given benchmark")
+    statsparser.add_argument(
+        "-i", "--id",
+        help="the ID or name of the controls file in the controls/ directory",
+        required=True)
+    statsparser.add_argument(
+        "-l", "--level",
+        help="the compliance target level to analyze",
+        required=True, choices=["low", "moderate", "high"])
```

Comment:
Not all controls use always high,medium, or low. For example in the CIS controls `l1_server` is used as a level. The script does have nice output if you use the wrong level. So this `choices` argument makes this script unusable for CIS profiles.

Reply:
Good catch. I'll remove the choices argument for now since providing that would be redundant with what's defined in the profile and what's generated at runtime.